pkgs/development/compilers/llvm/rocm/compiler-rt/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./compiler-rt-codesign.patch # Revert compiler-rt commit that makes codesign mandatory
-  ];
+  ] ++ [ ../../common/compiler-rt/libsanitizer-no-cyclades-rocm.patch ];
```

Comment:
The patch is quite large. Did you create this patch yourself? If not can we use fetchpatch?
Otherwise it seems that the chunksize is quite big when patching because it contains a lot of unrelated lines.

Reply:
After reading your comment, my question is answered:

This would be the preferred way:

```
(fetchpatch { url = "https://reviews.llvm.org/file/data/nrhbpc5axblqwx2xyyzv/PHID-FILE-wwcpjvquusomoddmqcwo/file"; sha256 = "<the actual hash>"; })
```